yarn.lock

```diff
@@ -1777,17 +1777,31 @@
     cross-fetch "^3.1.4"
     crypto-js "^4.1.1"
 
-"@psychedelic/plug-controller@0.23.4":
-  version "0.23.4"
-  resolved "https://npm.pkg.github.com/download/@Psychedelic/plug-controller/0.23.4/92ba5923065d37ea322cfc758861da0694959cfa#92ba5923065d37ea322cfc758861da0694959cfa"
-  integrity sha512-fwdlVPZidcHFt6x+o5OXKSyJi/VA7htkpZJ6UcU6WromEsBenaHKuRPdfJgJzRpY5AMWckgLKf2HVPL1fHxxJg==
+"@psychedelic/dab-js@1.5.0-beta.1":
```

Comment:
dab-js beta?

Reply:
That's from the controllers version

Reply:
yep, can you check if its ok to use that? why not a release?

Reply:
wil do